com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using UnityEditor.UIElements;
 using UnityEngine;
+using UnityEngine.Rendering;
```

Comment:
I would have prefered not having Rendering knowledge in the window as it just display what we give to it.

[optional] replace by: using RenderPipelineManager = UnityEngine.Rendering.RenderPipelineManager;

Reply:
Agreed. I'm mostly a C++ guy, so I did not know about aliases.